com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -96,6 +107,50 @@ public void initialized(@Nullable InitializedParams params) {
                     "copybooksWatcher", "workspace/didChangeWatchedFiles", createWatcher()),
                 new Registration("configurationChange", "workspace/didChangeConfiguration", null)));
     client.registerCapability(registrationParams);
+    try {
+      fetchSettings("broadcom-cobol-lsp.cpy-manager", null)
+          .thenAccept(
+              e -> {
+                JsonObject jsonObject = (JsonObject) e.get(0);
+                ConfigurationSettingsStorable configurationSettingsStorable =
+                    isValidJson(jsonObject);
+                ((SettingsProvider) settingsProvider).set(configurationSettingsStorable);
+              });
+    } catch (RuntimeException e) {
+      log.error(e.getMessage());
+    }
+  }
+
+  /**
+   * @param jsonObject - the object which comes from the client and contains configuration settings
+   * @return a custom object of type ConfigurableSettingsStorage if the JSON is valid or null if it
+   *     is failing the check
+   */
+  private ConfigurationSettingsStorable isValidJson(JsonObject jsonObject) {
```

Comment:
The name pattern 'is***', according to Java naming convention, used mostly for boolean methods. It would be more descriptive if you rename it so something like `parseSettings`